client/components/Overview/ProductCategory.jsx

```diff
@@ -12,9 +11,8 @@ function ProductCategory(props) {
   } else {
     return <div></div>
   }
-
 }
 
 export{
-  ProductCategory
+  ProductCategory,
```

Comment:
no need for a comma since this is the only export. It shouldnt break your code but just a note for the next commit